### PR TITLE
Adding claimReportingFees function

### PR DESCRIPTION
**Pull request by adrake33**
This is a new function designed to be used by the Portfolio: Reporting page to allow users to claim all of their reporting fees by clicking a single button.

### CODE REVIEW

**Comment by adrake33** [comment]
@bthaile Yes, I've been able to use the `estimateGas` flag to get it to return gas estimates.  I did see the error you mentioned earlier today, but then I realized the address in MetaMask had gotten changed to an address that didn't have any funds. Once I changed the address to the test address 0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb, it worked.

File: src/reporting/claim-reporting-fees.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          onFailed: function (error) {
+            errors[contract.address] = error;
+            nextContract();
+          },
+        }));
+        break;
+      case 1:
+        api().InitialReporter.redeem(assign({}, redeemPayload, {
+          "": p.redeemer,
+          tx: {
```
[the remark this answers is not in the document]
Comment (epheph) [reply src/reporting/claim-reporting-fees.js]: Sadly, solidity gets upset when you accept a named parameter (to satisfy an interface) but then don't use it. Seems like a solidity bug to me, but yea, that's the situation.

[the remark this answers is not in the document]
Comment (bthaile) [reply src/reporting/claim-reporting-fees.js]: I'm good with approving it. If no one else looks at it today I'll approve.

File: src/reporting/claim-reporting-fees.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      initialReporters: new BigNumber(0),
+      feeWindows: new BigNumber(0),
+      all: new BigNumber(0),
+    },
+  };
+  var errors = [];
+
+  async.eachLimit(p.redeemableContracts, PARALLEL_LIMIT, function (contract, nextContract) {
+    switch (contract.type) {
+      case 0:
```
Comment (epheph) [inline src/reporting/claim-reporting-fees.js]: Do we have constants for these contract types?

Comment (adrake33) [reply src/reporting/claim-reporting-fees.js]: Sounds good. I'll add this.

**Pull request merged**